Library/Homebrew/test/rubocops/components_order_spec.rb

```diff
@@ -90,6 +90,47 @@ class Foo < Formula
         end
       RUBY
     end
+
+    it ": the on_macos block is not after uses_from_macos" do
```

Comment:
Why the `: `? Please remove these. I commented on this before but you marked as resolved without actually making any changes...